donate/settings.py

```diff
@@ -70,6 +70,8 @@
 ALLOWED_HOSTS = env('ALLOWED_HOSTS')
 CSRF_TRUSTED_ORIGINS = ALLOWED_HOSTS
 USE_X_FORWARDED_HOST = env('USE_X_FORWARDED_HOST')
+CSRF_COOKIE_SECURE = True
+SESSION_COOKIE_SECURE = True
```

Comment:
Please also set `CSRF_COOKIE_SAMESITE` TO `'LAX'`
https://docs.djangoproject.com/en/2.2/ref/settings/#csrf-cookie-samesite

Reply:
@cadecairos `Lax` is the default anyway?

Reply:
Ah, yes, it is!